index.js

```diff
@@ -34,23 +34,23 @@ AnnotationPoller.prototype._installExtensions = function () {
 
 AnnotationPoller.prototype.start = function (loaded) {
   var _this = this
-  var updating = false
```

Comment:
Why remove the `updating` check? What happens if the interval runs while it's still waiting on a previous poll request?